### DISCUSSION TITLE
Metal Progression

### DISCUSSION THREAD

**Discussion by Strange-Bird-of-the-Moon** [Kenja-mc/shattered_stars#46] [Ideas]
### Preindustrial Age
Basic Furnace
- Copper - ore
- Zinc - ore
- Tin - ore
- Lead - ore
- Iron - ore
- Gold - ore
- Silver - ore

Alloy Furnace
- Brass - 2 copper + 1 zinc
- Bronze - 3 copper + 1 tin
- Electrum - 1 silver + 1 gold
- Wrought Iron - iron + slag

### Steam Age
Coke Oven/Ashery (likely split into multiple)
- Coke - coal/charcoal
- Ash - planks
- Quicklime (CaO, flux) - calcite (crushed?) or bonemeal?
- Lye (NaOH) - ash + quicklime + water
- Soap (if using Supplementaries) - lye + animal fat
- Potash (KOH, fertilizer) - ash + water
- Gunpowder - niter (saltpeter, KNO3) + sulfur + charcoal/coal

Blast Furnace
- Pig Iron - iron + coke
- Cast Iron - pig iron + flux
- Steel - pig iron + iron + flux

### Modern Age
Chemistry Station
- Sulfuric Acid (used for more efficient smelting/mining) - sulfur/slag + niter (saltpeter) + water + lead sheet (not consumed)
- Hydrochloric Acid - sulfuric acid + salt

Electrolytic Cell
- Nickel - ore
- Magnesium - ore/seawater + quicklime
- Aluminum - ore +  hydrochloric acid + flux
- Titanium - ore + magnesium + hydrochloric acid
- Beryllium - ore + magnesium + hydrochloric acid
- Tungsten -  ore + magnesium + hydrochloric acid (temporary, should probably have its own process)

Advanced Alloy/Blast Furnace
- Stainless Steel - steel + nickel
- Tool Steel - steel + tungsten + molybdenum?
- Beryllium Bronze - copper + beryllium
- Aluminum Bronze - copper + aluminum
- Constantan - copper + nickel 
- Invar - iron + nickel
- Nitinol - titanium + nickel
- Various combinations of magnesium, nickel, aluminum, titanium, and beryllium that don't have unique names

**Comment by Kenja-mc** [comment]
Everything under Advanced Alloy/Blast Furnace will probably go under a "Steel Age"

At some point we want to figure out which magic mod we want to use and where they lie in the tech tree as well.
